comptests/motr/test_motr_k8s_sanity.py

```diff
@@ -251,9 +233,28 @@ def test_cluster_shutdown_with_m0crate(self):
         config_file = os.path.join(os.getcwd(), "config/motr/test_29707_m0crate_workload.yaml")
         remote_file = TEMP_PATH + config_file.split("/")[-1]
         for node in self.motr_obj.get_node_pod_dict():
-            self.update_m0crate_config(config_file, node)
+            self.motr_obj.update_m0crate_config(config_file, node)
             self.motr_obj.m0crate_run(config_file, remote_file, node)
         self.motr_obj.shutdown_cluster()
         for node in self.motr_obj.get_node_pod_dict():
-            self.update_m0crate_config(config_file, node)
+            self.motr_obj.update_m0crate_config(config_file, node)
             self.motr_obj.m0crate_run(config_file, remote_file, node)
+
+    @pytest.mark.tags("TEST-29706", "TEST-29709")
+    @pytest.mark.motr_sanity
+    def test_cluster_shutdown_with_intrupted_motr_io(self):
+        """
+        This test will run the motr io on all the nodes in parallel and
+        verify the services after cluster shutdown
+        """
+        try:
+            for node in self.motr_obj.cortx_node_list:
+                node_process = multiprocessing.Process(target=self.motr_obj.run_motr_io,
+                    args=[node, [4], False, True])
+                node_process.start()
+        except (OSError, IOError, AssertionError) as exc:
```

Comment:
Do we want to raise exceptions in test file? @seagate-sarang-sawant

Reply:
I am not raising any exception here just catching it if occurs and ignoring it.

Reply:
Well the exception handling from process/thread does not work this way. You have to handle it within the  process run or get the exception from node_process  object.  However the real question pointed out by Sonal is what if your I/O fails and don't you want to flag the errors.